python/grass/jupyter/setup.py

```diff
@@ -21,7 +21,6 @@ def _set_notebook_defaults():
     """
     This function sets several GRASS environment variables that are
     important for GRASS to run smoothly in Jupyter.
-
     It also allows GRASS to overwrite existing maps of the same name.
     """
```

Comment:
This change is unrelated to this PR, but if you want to make your docstrings correct, you can follow [PEP 257 -- Docstring Conventions](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings):

```
def complex():
    """Form a complex number.

    More text. ...

    More text in more pragraphs. ...
    """
```

Reply:
To clarify here, I should open a new PR where I update all the docstrings to follow PEP 257? Or should I address this here?

Reply:
Fix the newly added docstrings. Leave the rest for later or for PRs where you are modifying the code anyway.

Reply:
I removed the changes in setup.py and reformatted to docstrings in interact-display.py to reflect PEP 257 (since these are the newly added docstrings).